devfiles/src/classes/options/SpeciesSelector.tsx

```diff
@@ -217,9 +242,28 @@ export default class SpeciesSelector extends InputOption {
                         <div className='form-check'>
                             {[...this.speciesMeta.groupByGroup.keys()].map((group) => (
                                 <div key={uuidv4()}>
+                                    <hr />
                                     <div className='speciesGroupRowDiv' key={uuidv4()}>
-                                        <hr />
                                         <strong>{group.value}</strong>
+                                        <button
+                                            className='btn btn-outline-dark btn-sm'
+                                            type='button'
+                                            onClick={() => {
+                                                [...this.speciesMeta.groupByGroup.get(group)].map(
+                                                    (s) => {
+                                                        this.selected.has(s)
+                                                            ? this.selected.delete(s)
+                                                            : this.selected.add(s);
+                                                    }
+                                                );
+                                                this.callback({
+                                                    checked: false,
+                                                    item: new Set([])
```

Comment:
this looks counterintuitive to me but I'll assume it does what you expect

Reply:
Empty set means don’t make changes in this case, it is correct, though yeah it’s not structured in a nice way